packages/wp-now/src/wp-now.ts

```diff
@@ -0,0 +1,168 @@
+import { NodePHP } from '@php-wasm/node'
+import { SupportedPHPVersion } from '@php-wasm/universal'
+import path from 'path'
+import { SQLITE_FILENAME, SQLITE_PATH, WORDPRESS_ZIPS_PATH } from './constants'
+import { downloadSqlite, downloadWordPress } from './download'
+
+interface WPNowOptions {
+  phpVersion?: SupportedPHPVersion
+  documentRoot?: string
+  absoluteUrl?: string
+}
+
+const DEFAULT_OPTIONS: WPNowOptions = {
+  phpVersion: '8.0',
+  documentRoot: '/var/www/html',
+  absoluteUrl: 'http://127.0.0.1:8881',
```

Comment:
this is building the default URL with _a_ default port number but it differs from what's in the constants module. maybe we can import those and build it programmatically here?

Reply:
Changed here to get the actual used port:

https://github.com/Automattic/wordpress-playground/blob/79f4b579dbf4c2f11ba13f77c0d6cff80e2f5b4a/packages/wp-now/src/wp-now.ts#L30-L33